tests/parsing/other/optionalPropValue.res

```diff
@@ -0,0 +1,18 @@
+module Parens = {
```

Comment:
I tried this.
First, module `Option` does not exist.
Second, using `Js.Option.map` this already gives a type error with the old compiler.
So even though this test has been checked in, I'm not sure what I'm looking at. What it's expected to do. And what it does instead.

Reply:
Playground link: https://rescript-lang.org/try?code=LYewJgrgNgpgBABQIYCcYDsDOcC8cDeAUHHAAJpIDGALgHSUjAAOI6G1xcs1cwSA1vDwAKAH4BLMDgD8ASlwA+ApxIAeMOIBuKknEkyiuo3rABaBQClMtAPJNq41rT5NhAD0Vw3snSQC+OgD0CpwBAUA